src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $mainColor;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: inherit;
+    font-weight: 500;
+    line-height: 18px;
+    color: $textColor;
+  }
+
+  &__code {
+    color: $secondaryColor;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-label {
+    display: flex;
+    align-items: center;
+    font-size: 10px;
+    line-height: 14px;
+    color: $textColor;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+
+  }
+
+  &__price-label {
+    line-height: 18px;
+    color: $secondaryColor;
+  }
+
+  &__price-value {
+    color: $textColor;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 12px 0;
+
+    background-color: $activeColor;
+    border-radius: 5px;
+    border: 1px solid $activeColor;
+
+    text-transform: uppercase;
+    color: $mainColor;
+    font-weight: 700;
+    text-decoration: none;
+
+    &:hover {
+      color: $activeColor;
+      background-color: $buttonBackground;
+      transition: $transitionTime;
```

Comment:
Let's move transition property to the main class, not on the hover state
And don't forget to specify the property, for which you want to apply the transition